packages/big-bang-core/README.md

```diff
@@ -18,58 +18,81 @@ This package deploys [Big Bang Core](https://repo1.dso.mil/platform-one/big-bang
 ### Pull down the code and binaries
 
 ```shell
-# clone the binaries
+# Clone the binaries
```

Comment:
Do you think we could use brew instructions for the install like here: https://zarf.dev/install/ ?

Reply:
Iron Bank doesn't support arm images, and m1 macs are arm-based. It is sort of positively I think to donut with Rosetta, but should be done with big caveats as it's not really supported.

Reply:
On the other note, yes let's remove it. That's an interpretation of how BB should work that we are actively working to replace anyway with things Tom is working on.

Reply:
@jeff-mccoy made those changes and a few other tweaks - hopefully in better shape for review now, thanks!